src/steamship/data/block.py

```diff
@@ -188,6 +199,59 @@ def is_video(self):
         """Return whether this is a video Block."""
         return self.mime_type in [MimeTypes.MP4_VIDEO, MimeTypes.WEBM_VIDEO]
 
+    @property
```

Comment:
Just want to actively +1 that I'm totally OK w/ us adding these kinds of helpers directly to Block